test/functional/api/autocmd_spec.lua

```diff
@@ -881,6 +881,93 @@ describe('autocmd api', function()
         eq([[:echo "Buffer"]], normalized_aus[1].command)
       end)
     end)
+
+    describe('id', function()
+      local _id, _id1
+      after_each(function()
+        api.nvim_del_autocmd(_id)
+        if _id1 then
+          api.nvim_del_autocmd(_id1)
+          _id1 = nil
```

Comment:
why is this needed? is clear() not called for each test?

Reply:
Aha thanks.